index.html

```diff
@@ -5,11 +5,13 @@
 <link rel="icon" href="favicon.ico" type="image/x-icon">
 <link rel="stylesheet" href="style.css">
 <link rel="stylesheet" href="awesomplete.css">
+<body onLoad="idiotCheck();">
 <div class="header">
 <a id="index">tetick.xyz</a>
 <a id="save">save</a>
 <a id="about-link">about</a>
 <a id="wrong-link">wrong!</a>
+<a id="idiot-link" title="who likes shiny big buttons with disgusting colours">i am an idiot!</a>
```

Comment:
I think the link can be "ugly!" and the alt title "i am an idiot who likes..." so it looks _friendlier_.